plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/plugin.properties

```diff
@@ -0,0 +1,2 @@
+pluginName = JAXB XJC Plug-in
```

Comment:
This plugin has the same pluginName as plugins/org.jboss.tools.locus.jaxb-xjc/2.2.7/plugin.properties, "JAXB XJC Plug-in". Should this one be different?